index.bs

```diff
@@ -6001,7 +6001,7 @@ To <dfn>serialize as a mapping</dfn> given |iterable|, |serialization options|,
 
 1. Let |serialized| be a new list.
 
-1. For |item| in |iterable|:
+1. For |item| in [=IteratorToList=]([=GetIterator=](|iterable|)):
```

Comment:
[The latest version of ECMA262's GetIterator abstract operation](https://tc39.es/ecma262/#sec-getiterator) requires two arguments--we'll want to specify "sync" in this case.